File: mario.py
from cs50 import get_int

# Prompt for user input untill input is in the range of 1 to 8
while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

# Outer loop changes row
for temp in range(height):

    # Inner loops to print out individual rows

    # Prints initial spaces
    for i in range(temp, height - 1):
        print(" ", end="")

    # Prints initial #
    for i in range(temp+1):
        print("#", end="")

    # Gives indent
    print("  ", end="")

    # Prints final #
    for i in range(temp+1):
        print("#", end="")

    # Print new line after each row
    print()

File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    //Prompt user input
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    //Outer loop changes row
    for (int temp = 0; temp < height; temp++)
    {
        //Inner loops to print out individual rows

        //Prints initial spaces
        for (int i = temp; i < height - 1; i++)
        {
            printf(" ");
        }

        //Prints initial #
        for (int i = 0; i <= temp; i++)
        {
            printf("#");
        }

        //Gives indent
        printf("  ");

        //Prints final #
        for (int i = 0; i <= temp; i++)
        {
            printf("#");
        }

        printf("\n");
    }
}